explorer/client/src/pages/config/AppRoutes.tsx

```diff
@@ -17,6 +18,7 @@ const AppRoutes = () => {
             <Route path="/transactions/:id" element={<TransactionResult />} />
             <Route path="/addresses/:id" element={<AddressResult />} />
             <Route path="/missing/:id" element={<MissingResource />} />
```

Comment:
This PR means we now have 2 error pages when an incorrect search is made. Could we rationalise this to one? For example, we could remove MissingResource.

Reply:
it's 2 different errors.
this new one is that the search itself is malformed.

We can combine them if we want, i just didn't want to have to rework the missing page into a more generic error

Reply:
Combining the two makes sense as the presentation and layout will be similar.